STM32/AC/Core/Src/ACBoard.c

```diff
@@ -182,22 +184,39 @@ static void CAallOn(bool isOn)
 }
 static void CAportState(int port, bool state, int percent, int duration)
 {
+	uint8_t pwmPercent;
+	getPWMPinPercent(port-1, &pwmPercent);
+	// If heat sink has reached the maximum allowed temperature and user
+	// tries to heat the system further up then disregard the input command
+	if (heatSinkTemperature > MAX_TEMPERATURE && percent > pwmPercent)
```

Comment:
how fast is the temperature of the switchboard reacting to the adjustments it makes?

asking because if its very slow, it sounds like here we would accept the off command of the LoopControl's heater control cycle and then reject the on commands until the switchboard is at the right temperature. So if its slow, the pwm adjust down is only used in the first LoopControl's heater cycle and then it remains off until it gets to the switchboard temperature.

I guess that *if* LoopControl's have not been set a max power *and* the switchboard oveheat happens far from the oven target then the pwm adjustments is likely to rule most of the time. Otherwise the behavior I mentioned above is likely to happen, but only if the switchboard temperature reaction is very slow. 

ps. remember for anyone changing from 20-30% to 100% power to carefully review their physical's system sizing (power cable, fuses) + their pcontrol values in IO.conf (they could see bigger overshoot as max power plays a part on slowing down the rising temperature).